config.xml

```diff
@@ -22,11 +20,11 @@
   <preference name="BackupWebStorage" value="none"/>
   <preference name="SplashScreen" value="screen"/>
   <preference name="SplashScreenDelay" value="100"/>
-  <preference name="xwalkVersion" value="19+" />
-  <preference name="xwalkCommandLine" value="--disable-pull-to-refresh-effect" />
-  <preference name="xwalkMode" value="embedded" />
-  <preference name="xwalkMultipleApk" value="true" />
-  <preference name="emSensorDataCollectionProtocolApprovalDate" value="2016-07-14" />
+  <preference name="xwalkVersion" value="19+"/>
+  <preference name="xwalkCommandLine" value="--disable-pull-to-refresh-effect"/>
+  <preference name="xwalkMode" value="embedded"/>
+  <preference name="xwalkMultipleApk" value="true"/>
+  <preference name="emSensorDataCollectionProtocolApprovalDate" value="2016-07-14"/>
```

Comment:
Are all of these real changes? It looks like they are primarily related to removing whitespace...